recipes/rse/meta.yaml

```diff
@@ -0,0 +1,114 @@
+{% set name = "rse" %}
+{% set version = "0.0.30" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2f27a8749cd51636ef1877a29ab2c5db91c4d71560f33cb71cd9f90af45e74ce
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - rse=rse.client:main
+  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - pytest-runner
+  run:
+    - python >=3
+    - sqlalchemy
+    - requests
+    - flask
+    - flask-socketio
+    - flask-restful
+    - gevent-websocket
+    - beautifulsoup4
+    - html5lib
+
```

Comment:
I believe it would be nice to add these dependencies to your ``setup.py`` as well
Because according to your metadata it just requires ``requests`` at runtime

Reply:
These are in extra_requires, the reason being that you can use the simplest version of the package without them (e.g., you don't have to use the flask interface). The reason I have them here (but they aren't in setup.py) is because the user typically expects a conda install to be complete and have everything. Is that ok?

Reply:
That is ok, but that is just a bit odd when you compare with the package on pypi which will not install those things
but that is fine then :)